index.html

```diff
@@ -0,0 +1,161 @@
+    <p class="paragraph">We Believe We Can End the AIDS Epidemic. Since 2021, AIDS-Related Deaths Have Dropped by 35% in the Part of the World Where We Work. Rated a Four Star Charity. 900+ Employees Worldwide. 100% Tax Deductible. Help Us Stand #ByHerSide.
+    </p>
+    <div class="back-imag"></div>
+</div>
+<div class="event-day">
+    <h4>2022.10.15(THU) ~ 16(FRI)</h4>
+    <p>@ Kololo National Stadium, Kampala</p>
+</div>
+ <section class="background-image">
+        <div class="main"><h3>Main Program</h3></div>
+        <div class="horizontal-lines">
+            <hr>
+        </div>
+        <div class="sec">
+            <div class="main-section">
+                <div class="exibition">
+                    <img src="assets/images/roulette.png" alt="">
+                    <h3>Leacture</h3>
+                    <p>December 1st kicks off AIDS Awareness Month with World AIDS Day. Leading up to the month of December was the 40th Anniversary</p>
+                </div>
+                <div class="exibition">
+                    <img src="assets/images/whiteboard.png" alt="">
+                    <h3>CC Exhibition</h3>
+                    <p>With HIV related products and services on display in interactive 3D booths; chat functions enabling real-time dialogue with exhibitors</p>
+                </div>
+                <div class="exibition">
+                    <img src="assets/images/chat.png" class="forum1" alt="">
+                    <h3>Forum</h3>
+                    <p>The Forum HIV Prevention Trial Design Project convenes a series of public meetings.</p>
+                </div>
+                <div class="workshop">
+                    <img src="assets/images/teacher.png" alt="">
+                    <h3>Workshop</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="workshop">
+                    <div class="redlogo-container">
+                        <img src="assets/images/download.png" class="red-logo" alt="">
+                    </div>
+                    <h3>CC Ignite</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="global">
+                    <h4 class="bar">Join HIV Global Summit 2022</h4>
+                </div>
+            </div>
+        </div>
+        <div   class="program">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </div>
+</section>
+<div id="speaker-list"></div>
+<h2 class="speaker-list">Featured Speakers</h2>
+<div class="center-hor">
+    <div class="horizontal-line"></div>
+</div>
+<div id="speaker-lists"></div>
+<footer>
+    <div class="footer">
+        <h2>Partner</h2>
+        <div class="sub-section">
+            <div class="footer-section">
+                <ul class="lists">
+                    <li>Have Questions ?</li>
```

Comment:
- [ ] As requested by the previous reviewer, Please fix the design for the partner section and make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169514530-cddb6c63-396f-4aff-bffc-b427062577bd.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169514657-46496754-59d0-406b-9059-4ad40baf06aa.png)